WordPress/Classes/Utility/Media/ImageLoader.swift

```diff
@@ -242,7 +250,7 @@ import MobileCoreServices
             }
 
             if self.imageView.shouldShowLoadingIndicator {
-                self.loadingIndicator.state = .error
+                self.stopLoadingAnimation()
```

Comment:
⚠️ I'm not quite sure of this change. Would you mind explaining why this has to be done? 

I think it's causing a regression in parts where `CachedAnimatedImageView` is used. Notice the gray background color is not being shown when using this branch:

`develop` | This branch
--------|-------
![IMG_0389](https://user-images.githubusercontent.com/198826/66498549-7ff2b400-ea7b-11e9-8f50-78692868ecaf.PNG)        |       ![IMG_0390](https://user-images.githubusercontent.com/198826/66498548-7f5a1d80-ea7b-11e9-80bf-3eb2520243d0.PNG)

Reply:
My reading of the code is that the `error` state is used in conjunction with the `errorView`. Since we're not using the error view, and I'd noticed that the loading indicator was not removed from the hierarchy when stopped, I thought it should probably be removed when the retry view is shown.